package.json

```diff
@@ -7,14 +7,15 @@
     "url": "https://github.com/AdobeDocs/commerce-webapi"
   },
   "dependencies": {
-    "@adobe/gatsby-theme-aio": "4.6.6",
+    "@adobe/gatsby-theme-aio": "^4.7.9",
```

Comment:
We haven't bumped the theme yeet in any other repos. I think @bdenham mentioned that this version throws some JS console errors. I recommend backing this out unless it's necessary for this PR.

Reply:
Just wanted to enable this fix: https://jira.corp.adobe.com/browse/DEVSITE-600

Reply:
Ah, that's necessary for this PR. Thanks for confirming.

Reply:
@dshevtsov, if you need the fix for frameSrc then I think we must bump the version. But I would pin it to 4.7.9.

Reply:
We still can use the absolute link until all the tests are complete to be sure 4.7.9 won't do any harm.